Makefile

```diff
@@ -61,7 +61,10 @@ clean-docker:
 	docker rmi -f $$(docker images -q $(COMPANY_NAME_LOW)/*) || exit 0
 
 deploy: $(DOCKER_TARGETS)
-	$(foreach TARGET,$(DOCKER_TARGETS),docker push $(subst $(COLON),:,$(TARGET));)
+	$(foreach TARGET,$(DOCKER_TARGETS), \
+		for i in {1..10}; do \
+			docker push $(subst $(COLON),:,$(TARGET)) && break; \
```

Comment:
Maybe add some timeouts between retries?
I think if push failed if something wrong with network - it's a good idea to wait for minute or two and try again
And in that case I think 3 retries is enough and if not pushed - some major problem is exists

Reply:
A minute is probably way to high. 10 seconds should be enough cause if it fails twice it will likely fail a third time and this would add 3 minutes where the pipeline is just waiting for nothing.